compiler/stmt.py

```diff
@@ -748,6 +925,19 @@ def _import_native(self, name, values):
                                       util.go_str(name), members.expr)
     return mod
 
+  def _import_wildcard(self, node):
+    """If __all__ is defined, attempts to bind everything from it,
+    falls back on __dict__ otherwise.
+
+    Note: items in __dict__ whose names start with '_' are skipped.
+    """
+    module_name = node.module
+
+    with self.block.alloc_temp() as members, \
```

Comment:
IIUC members is a temporary that will be filled up, used to populate the globals dict and then thrown away. I would just populate the globals dict as I iterate over the imported module's members and avoid the temporary.